### PR TITLE
Tiny clean up optimizer package.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
@coocood  PTAL

File: optimizer/validator.go [hunk 1]
```diff
@@ -113,7 +113,7 @@ func (v *validator) checkSameColumns(exprs ...ast.ExprNode) {
 	}
 }
 
-// checkFieldList checks there is only one '*" and each field has only one column,
+// checkFieldList checks there is only one '*" and each field has only one column.
```
Comment (shenli) [inline optimizer/validator.go]: Grammar error.
checkFieldList checks if there is only one '*" and each field has only one column.

**Comment by shenli** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**